src/common/errors.js

```diff
@@ -1,5 +1,22 @@
-class ConfigError extends Error {}
-class UndefinedParserError extends Error {}
-class ArgExpansionBailout extends Error {}
+class ConfigError extends Error {
+  constructor(...args) {
+    super(...args);
+    this.name = this.constructor.name;
+  }
+}
```

Comment:
👍 For this change, but should use property instead.

```
class ConfigError extends Error {
  name: "ConfigError"
}
```

Reply:
There seems another error missing name https://github.com/prettier/prettier/blob/fc6419ab63a8b3bc2b4458bb5771fe64638ab013/src/cli/format.js#L21

Reply:
Added in [`DebugError`](https://github.com/prettier/prettier/pull/14913/files#diff-603b6977dacabdf21d2dc3bfa189109d6a3416f903992096deb1024bae92c1fbR21-R23).